client/src/main/java/io/split/client/LocalhostSegmentChangeFetcher.java

```diff
@@ -24,8 +24,9 @@ public LocalhostSegmentChangeFetcher(String filePath){
     @Override
     public SegmentChange fetch(String segmentName, long changesSinceThisChangeNumber, FetchOptions options) {
         try {
-            JsonReader jsonReader = new JsonReader(new FileReader(String.format("%s/%s.json", _file, segmentName)));
-            return Json.fromJson(jsonReader, SegmentChange.class);
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
JsonReader jsonReader = new JsonReader(new FileReader(String.format("%s/%s.json", _file, segmentName)));
            SegmentChange segmentChange = Json.fromJson(jsonReader, SegmentChange.class);
            return LocalhostSanitizer.sanitization(segmentChange);

Reply:
In this way, it's not going to work. If an element isn't in the JSON file is going to throw an exception.